### DISCUSSION TITLE
YAML for RSS type: partial

### DISCUSSION THREAD

**Discussion by davidleitch1** [quarto-dev/quarto-cli#9279] [Q&A] [answered]
### Description
I would like to know the correct YAML to set the rss feed to partial. The following code results in an indentation error at the line "type: partial". So far despite lots of fiddling I cannot get the correct YAML. TIA
````


---
title: "Articles "
listing:
  contents: posts
  sort: "date desc"
  type: default
  categories: true
  sort-ui: false
  filter-ui: false
  feed: true
    type: partial 
page-layout: article
title-block-banner: false
---

**Comment by mcanouil** [comment, marked as answer]
In YAML, you cannot have a key both as parent and has a value.

The following will always be invalid, thus the suggestion to remove the indentation of the `child: value`.

```yaml
key: value
  child: value
```

When a key has subkeys, it has no atomic value:

```yaml
key:
  child: value
```

**Reply by davidleitch1** [reply]
Mickaeil:  My thanks.
